tt_metal/llrt/tt_cluster.hpp

```diff
@@ -265,7 +255,7 @@ class Cluster {
 
     // There is one device driver per PCIe card. This map points id of the MMIO device points to the associated device
     // driver
-    std::unordered_map<chip_id_t, std::unique_ptr<tt_device>> mmio_device_id_to_driver_;
+    std::unique_ptr<tt_device> driver_;
```

Comment:
Nit: Can you update the comment above to specify that there is one driver for all devices in the cluster